pyccl/neutrinos.py

```diff
@@ -11,7 +12,8 @@
 }
 
 
-def Omeganuh2(a, m_nu, T_CMB=None):
+@warn_api()
+def Omega_nu_h2(a, *, m_nu, T_CMB=None):
```

Comment:
Replied above:
If you're asking why am I renaming this, it's because I find `Omeganuh2` extremely hard to read. Python universally adopts `snake_case` while C-like languages adopt `SwapCase`. Similar to how we have `halomod_power_spectrum` rather than `HalomodPowerSpectrum`.